src/components/complete-profile-fourth/hooks.ts

```diff
@@ -0,0 +1,172 @@
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
```

Comment:
I think this hook should be stored in hooks folder

Reply:
I'm not sure that in this case we shoud make hooks folder as we have just one hook